### ISSUE TITLE
Padding Bug in CMS Editor

### ISSUE THREAD

**Issue by Marmik2003** [wagtail/wagtail#8134]
<img width="956" alt="Screenshot 2022-03-16 at 13 34 08" src="https://user-images.githubusercontent.com/44599895/158545425-d1b7a764-dc46-41cb-b70b-92583a1be1e3.png">

Search logo is not aligned, also right button and left text is not vertically aligned.

**Comment by thibaudcolas** [comment]
Hi @Marmik2003, could you take a moment to fill our whole bug report template? Otherwise it’s hard to figure out whether this is an issue with your setup or not. Here is the template:

```
Found a bug? Please fill out the sections below. 👍

### Issue Summary

A summary of the issue.


### Steps to Reproduce

1. (for example) Start a new project with `wagtail start myproject`
2. Edit models.py as follows...
3. ...

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

* I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: (yes / no)


### Technical details

* Python version: Run `python --version`.
* Django version: Look in your requirements.txt, or run `pip show django | grep Version`.
* Wagtail version: Look at the bottom of the Settings menu in the Wagtail admin, or run `pip show wagtail | grep Version:`.
* Browser version: You can use https://www.whatsmybrowser.org/ to find this out.
```

**Comment by Marmik2003** [comment]
Found a bug? Please fill out the sections below. 👍

### Issue Summary
Search logo is not aligned, also right button and left text is not vertically aligned.


### Steps to Reproduce

1. Start a new project with `wagtail start myproject`
2. create su
3. open cms document section 

* I have confirmed that this issue can be reproduced as described on a fresh Wagtail project: yes

### Technical details

* Python version: 3.10.0
* Django version: 4.0.1
* Wagtail version: v2.16.1
* Browser version: Chrome 99 (M1 Pro MacBook Pro).

**Comment by PaarthAgarwal** [comment]
I noticed the search logo misalignment as well but I didn't understand what you mean by "right button" and "left text". Please explain.

**Comment by Marmik2003** [comment]
<img width="801" alt="image" src="https://user-images.githubusercontent.com/44599895/159110274-913736d8-6130-48fc-b4cd-37ed8ec06977.png">
They are not vertically aligned to center, also below width of 800px, the plus icon seems ugly

**Comment by PaarthAgarwal** [comment]
We should fix the plus icon indeed. The search logo needs some validation if it should be kept like this or aligned with the text.

**Comment by Marmik2003** [comment]
I'll fix it

**Comment by Marmik2003** [comment]
<img width="800" alt="image" src="https://user-images.githubusercontent.com/44599895/159113363-3366b64b-9301-4405-bbd2-01fd5546acef.png">
fixed

**Comment by talkshrey** [comment]
Has the issue been fixed and closed or can it be assigned to me?

**Comment by lb-** [comment]
I cannot see a PR up yet linked to this issue so go for it @talkshrey 

@Marmik2003 when you say 'fixed' in the comment above, what did you mean? Do you have some rough CSS you can put in a comment that fixed it.

**Comment by Marmik2003** [comment]
Will create a PR today.

**Comment by Marmik2003** [comment]
<img width="689" alt="image" src="https://user-images.githubusercontent.com/44599895/159233709-418904f4-5aaa-4d6c-badf-ae6f8b6c5886.png">

I have  written this code to solve this issue, but it is showing max nesting depth error while running test because of media queries.
`Expected nesting depth to be no more than 3  max-nesting-depth `

Any idea to solve this?

**Comment by PaarthAgarwal** [comment]
Unable to replicate the mentioned behavior for the plus icon in the top right corner.
The plus icon appears normal to me as shown in your https://github.com/wagtail/wagtail/issues/8134#issue-1170683870.

**Comment by zerolab** [comment]
related #[7863](https://github.com/wagtail/wagtail/issues/7863)

**Comment by yash112-lang** [comment]
Hey **@thibaudcolas**, is this issue is **still open**? Can I work on it?

**Comment by Marmik2003** [comment]
Yes, will create a PR today that will solve the issue with suggestions of
@stevedya

**Comment by thibaudcolas** [comment]
Fixed in #8310.
